### DISCUSSION TITLE
April 2020: worked on a webmail and a bit of OpenSMTPD too · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#119] [Articles]

**Comment by ngortheone** [comment]
@poolpOrg 
unless this is a word play that I didn't understand
>  I still want to itch that scratch that annoys me since then.

you probably wanted to say "scratch that itch"

**Comment by ngortheone** [comment]
Thanks for the cool update. There is another interesting webmail project  that @emersion  is working on [koushin](https://git.sr.ht/~emersion/koushin), but as far as I know it is written in go

**Comment by myfirstnameispaul** [comment]
Is the webmail project code up anywhere?

**Comment by poolpOrg** [comment]
> you probably wanted to say "scratch that itch"

yes, fixed :-)

@myfirstnameispaul as written:

> Right now, having no experience with react causes me to do a lot of write and throw-away code, there’s no point in sharing code at this point.

**Comment by bzhpwr** [comment]
Thank U for all your work and post. 
OpenSMTP is so nice, so simple and efficient.
Leaving in Rennes, hop I can meet U one day.

**Comment by cypa** [comment]
"I want to be able to read my mail from various places and not rely on a heavy client."
don't you see the web-browser is one of the heaviest clients ever?

**Comment by poolpOrg** [comment]
> "I want to be able to read my mail from various places and not rely on a heavy client."
> don't you see the web-browser is one of the heaviest clients ever?

https://en.wikipedia.org/wiki/Rich_client
